### PR TITLE
RBAC: Add test for create clusterrolebindding

### PR DESCRIPTION
**Pull request by zjj2wry**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by dims** [comment]
@k8s-bot ok to test

**Comment by zjj2wry** [comment]
@dims i execute hack/make-rules/../../hack/verify-bazel.sh, so test restart. but code not change :)

**Comment by zjj2wry** [comment]
@mengqiy can you help me review this? thanks

File: hack/make-rules/test-cmd-util.sh [hunk 1]
```diff
@@ -3341,31 +3341,39 @@ run_clusterroles_tests() {
...
 
-  # test `kubectl create rolebinding/clusterrolebinding`
-  # test `kubectl set subject rolebinding/clusterrolebinding`
+  # test `kubectl create clusterrolebinding`
+  # test `kubectl set subject clusterrolebinding`
   kubectl create "${kube_flags[@]}" clusterrolebinding super-admin --clusterrole=admin --user=super-admin
   kube::test::get_object_assert clusterrolebinding/super-admin "{{range.subjects}}{{.name}}:{{end}}" 'super-admin:'
   kubectl set subject "${kube_flags[@]}" clusterrolebinding super-admin --user=foo
   kube::test::get_object_assert clusterrolebinding/super-admin "{{range.subjects}}{{.name}}:{{end}}" 'super-admin:foo:'
+  kubectl create "${kube_flags[@]}" clusterrolebinding mult-user --clusterrole=admin --user=user-1 --user=user-2
```
Comment (mengqiy) [inline hack/make-rules/test-cmd-util.sh]: s/mult-user/multi-users

File: hack/make-rules/test-cmd-util.sh [hunk 2]
```diff
@@ -3341,31 +3341,39 @@ run_clusterroles_tests() {
...
+  kube::test::get_object_assert clusterrolebinding/mult-user "{{range.subjects}}{{.name}}:{{end}}" 'user-1:user-2:'
 
   kubectl create "${kube_flags[@]}" clusterrolebinding super-group --clusterrole=admin --group=the-group
   kube::test::get_object_assert clusterrolebinding/super-group "{{range.subjects}}{{.name}}:{{end}}" 'the-group:'
   kubectl set subject "${kube_flags[@]}" clusterrolebinding super-group --group=foo
   kube::test::get_object_assert clusterrolebinding/super-group "{{range.subjects}}{{.name}}:{{end}}" 'the-group:foo:'
+  kubectl create "${kube_flags[@]}" clusterrolebinding mult-group --clusterrole=admin --group=group-1 --group=group-2
```
Comment (mengqiy) [inline hack/make-rules/test-cmd-util.sh]: multi-groups

**Comment by mengqiy** [comment]
@zjj2wry Hint: https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/47039/pull-kubernetes-unit/47812/artifacts/junit_test-cmd.xml

**Comment by mengqiy** [comment]
/assign @pwittrock 
for approval

**Comment by zjj2wry** [comment]
@pwittrock this is all green. mind for approve this :) , thanks

**Comment by zjj2wry** [comment]
@mengqiy @pwittrock i rebase master fix test failed. - -

**Comment by mengqiy** [comment]
You don't need to rebase on HEAD.
Presubmit always apply your change on latest HEAD.

**Comment by zjj2wry** [comment]
cc @kubernetes/sig-cli-pr-approver

**Comment by zjj2wry** [comment]
@fabianofranz @pwittrock mind approve this? thanks

**Comment by zjj2wry** [comment]
@pwittrock this need your approve. thanks

**Comment by zjj2wry** [comment]
from v1alpha1 to v1beta1~

**Comment by apelisse** [comment]
Thanks!

/lgtm

**Comment by apelisse** [comment]
/retest

That looks fishy

**Pull request merged**